cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.update import update
+
+pytestmark = [
+    pytest.mark.provider([RHEVMProvider], scope="module")
```

Comment:
currently in tagging the user for quota, the minimum value is `10GB`, and with vsphere the default template `damnsmalllinux`, which would fail quota test as we're trying to provision over the quota limit

Reply:
@psav, ^^

@vatsalparekh, I guess we need to ask provider owners to add small templates to provider.

Reply:
@izapolsk `damnsmalllinux` is already there for small use, we need option for a big template.